src/types/onyx/Session.ts

```diff
@@ -12,6 +14,8 @@ type Session = {
     accountID?: number;
 
     autoAuthState?: string;
+    /** Server side errors keyed by microtime */
+    errors?: OnyxCommon.Errors;
```

Comment:
maybe we can only import `Errors` from this common namespace?

Reply:
Everywhere we are are importing it as OnyxCommon.Errors so I would follow this pattern 😄